.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Personally, I prefer not to include specific tool configurations in version control unless it has been agreed that all folks working on the code should use that tool / editor.

Reply:
Team Bama uses VS Code as a standard, so I'd rather keep it, at least for now.